cumulusci/tasks/bulkdata/step.py

```diff
@@ -9,13 +9,13 @@
 from enum import Enum
 from typing import Any, Dict, List, NamedTuple, Optional
 
-import lxml.etree as ET
 import requests
 
 from cumulusci.core.exceptions import BulkDataException
 from cumulusci.core.utils import process_bool_arg
 from cumulusci.tasks.bulkdata.utils import iterate_in_chunks
 from cumulusci.utils.classutils import namedtuple_as_simple_dict
+from cumulusci.utils.xml import elementtree_parse_string
```

Comment:
You've replaced lxml with xml.etree here

Reply:
@prescod hmm. What's the motivation? The two etrees are not exactly the same and my impression has been that lxml provides a more ergonomic and complete API, with an added bonus of speed. I think I'd need to take a much closer look at the differences and at whether we are exposing lxml internals anywhere that others might have built on.

In any case it feels like a slightly riskier change and might be good to do separately from this one.

Reply:
I'll back it out, but I think that using two different APIs is just slightly confusing, it's another dependency (a binary one!) and so forth.

Based just on guesstimating the implementation details it isn't clear that lxml is faster, actually. They are both based on c-libraries underneath and I think that lxml's data structures are more elaborate to support more sophisticated use cases, so it may actually be slower. The LXML docs kind of imply it often is.

But either way it's a pretty low priority. and I was on the fence about whether to take the risk, so I can back it out.

If you agree that we should move towards the simpler, batteries-included API, I'll make a note like that in the docs so that people know to pick the simpler one if they can.